tests/supervisor/views/test_operator.py

```diff
@@ -350,7 +350,7 @@ async def test_websocket_logs_invalid_auth(aiohttp_client, mocker):
     response = await websocket.receive()
     # Subject to change in the future, for now the connexion si broken and closed
     assert response.type == aiohttp.WSMsgType.TEXT
-    assert response.data == '{"status": "failed", "reason": "string indices must be integers"}'
+    assert response.data == '{"status": "failed", "reason": "Invalid format for auth packet"}'
```

Comment:
You should probably provide a link or information about the expected format and what error is here for way easier debugging

Reply:
done. It was actually not documented so I added a whole documentation for it